client/blocks/qr-code-login/style.scss

```diff
@@ -0,0 +1,83 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+.qr-code-login {
+	display: flex;
+	justify-content: space-between;
+	flex-direction: column;
+
+	@include break-small {
+		flex-direction: row;
+	}
+
+	canvas {
+		max-width: 100%;
+		margin-bottom: 20px;
+        height: auto !important;
```

Comment:
Why do we need the `!important`?

Reply:
I was able to refactor the code so that is no longer needed.